app/services/idv/steps/verify_base_step.rb

```diff
@@ -226,6 +226,13 @@ def process_async_state(current_async_state)
           delete_async
           mark_step_incomplete(:verify)
           @flow.analytics.idv_proofing_resolution_result_missing
+          log_idv_verification_submitted_event(
+            success: false,
+            failure_reason: {
+              timeout:
+                ['We are experiencing higher than usual wait time processing your request.'],
```

Comment:
I'd almost say just use the slug `idv.failure.timeout`, because if we sue `I18n.t(...)` we'll get 3 different strings potentially (one for each locale)

Reply:
Yeah correct

Reply:
what about this? failure_reason: { idv_verification: [:timeout] }

Reply:
Love the simplification, yes!